consensus/src/network_interface.rs

```diff
@@ -83,14 +86,17 @@ pub enum ConsensusMsg {
     OrderVoteMsg(Box<OrderVoteMsg>),
     /// RoundTimeoutMsg is broadcasted by a validator once it decides to timeout the current round.
     RoundTimeoutMsg(Box<RoundTimeoutMsg>),
+    /// RPC to get a chain of block of the given length starting from the given block id, using epoch and round.
+    BlockRetrievalRequestV2(Box<BlockRetrievalRequest>),
```

Comment:
Same as for RPC request. V0 for the original variant and drop the V2 for the new variant?

Reply:
Changed to 

```rust
#[derive(Debug)]
pub enum IncomingRpcRequest {
    /// NOTE: This is being phased out in two releases to accommodate `IncomingBlockRetrievalRequestV2`
    /// TODO @bchocho @hariria can remove after all nodes upgrade to release with enum BlockRetrievalRequest (not struct)
    DeprecatedBlockRetrieval(DeprecatedIncomingBlockRetrievalRequest),
    BatchRetrieval(IncomingBatchRetrievalRequest),
    DAGRequest(IncomingDAGRequest),
    CommitRequest(IncomingCommitRequest),
    RandGenRequest(IncomingRandGenRequest),
    BlockRetrieval(IncomingBlockRetrievalRequest),
}
```